testsuite/smoke-test/src/client.rs

```diff
@@ -14,27 +15,20 @@ use libra_types::{
 };
 
 #[test]
-fn smoke_test_multi_node() {
-    let (_swarm, client_proxy) = setup_swarm_and_client_proxy(4, 0);
-    test_smoke_script(client_proxy);
-}
-
-#[test]
-fn smoke_test_single_node() {
-    let (_swarm, client_proxy) = setup_swarm_and_client_proxy(1, 0);
-    test_smoke_script(client_proxy);
+fn test_create_mint_transfer() {
+    let (_env, client) = setup_swarm_and_client_proxy(1, 0);
+    check_create_mint_transfer(client);
 }
 
 // Test if we commit not only user transactions but also block metadata transactions,
 // assert committed version > # of user transactions
 #[test]
-fn smoke_test_single_node_block_metadata() {
-    let (swarm, mut client_proxy) = setup_swarm_and_client_proxy(1, 0);
-    // just need an address to get the latest version
+fn test_create_mint_transfer_block_metadata() {
+    let (env, mut client) = setup_swarm_and_client_proxy(1, 0);
+    check_create_mint_transfer(env.get_validator_client(0, None));
+
     let address = AccountAddress::from_hex_literal("0xA550C18").unwrap();
-    // this script does 4 transactions
-    test_smoke_script(swarm.get_validator_client(0, None));
-    let (_account, version) = client_proxy
+    let (_account, version) = client
```

Comment:
but now you have a magic 4 for version below....

Reply:
frankly I don't see thae value of that check below (40-44) you would have failed in the function call, but if you're using this function call more than once, we should just kill this test.

Reply:
Yeah... I felt the test was a bit unnecessary, too. What I've done is added the comment back into the test and merged the two overlapping smoke tests to avoid the redundancy.